_pages/events/winter-school.md

```diff
@@ -24,7 +37,7 @@ Do you enjoy being outdoors but don't know what to do when the temperature drops
 
 Winter School is an annual IAP course presented by the members of MITOC. Its goal is to teach the skills one needs to enjoy outdoor sports in the winter. During the month of January, participants attend lectures every Tuesday and Thursday evening and spend time outdoors during the weekends. Trips are offered at various levels of intensity and include hiking, snowshoeing, cross-country skiing, back-country skiing, ice climbing, winter camping, orienteering, and mountaineering.
 
-### What is the format?
+<!-- ### What is the format?
```

Comment:
:*(

(We could also delete in lieu of commenting out, since Git history preserves it)

Reply:
I'd generally delete commented code, but I'm pretty sure we'll want to bring it back as it is next year, and I don't want to make it harder for whoever is going to bring the page back to normal ;)

Reply:
Makes perfect sense!

And I know many editors aren't frequent Git users, so it's probably easiest for this content to be readily available.

I look forward to uncommenting this in the future!!